### DISCUSSION TITLE
Cloudflare proxy as subdirectory setup correctly, but blog is catching all requests

### DISCUSSION THREAD

**Discussion by slightlyannoyed1** [plausible/analytics#2124] [Anything Else]
I followed the setup guide for proxying through Cloudflare, thanks!

I'm stuck though because when I try to access the URL, my ghost blog will always catch it and show 404.

If I run the test on a proxy worker, it shows valid 200 result (screenshot attached).

What am I missing here?  

Here is the working workers.dev URL: https://soft-moon-d1db.streamable1.workers.dev/
Here is my subdirectory url that doesn't seem to work outside of cloudflare : https://todotemplates.com/starlight/script.js
my route is setup as: todotemplates.com/starlight/*
![image](https://user-images.githubusercontent.com/4968795/185792047-aca43842-3f71-4045-996b-7e8bb100e27d.png)

**Comment by metmarkosaric** [comment]
Hi @jeffreywyman! I just checked https://todotemplates.com/starlight/script.js and it loads all fine here. Seems like you figured it out?

**Reply by slightlyannoyed1** [reply]
Thanks for checking!  Am I crazy or does it actually load the script?  It shows a 404 page for me: 
![image](https://user-images.githubusercontent.com/4968795/185798447-0e8ec944-8338-41e4-85ef-2604dce397bb.png)

**Reply by metmarkosaric** [reply]
For me it loads all fine. Try to clear your cache please.

**Comment by slightlyannoyed1** [comment]
For some reason clearing my cache did not help, but you are right, it does
work when I try a different browser. Not sure what the deal is there, but
I'm okay with it. Thank you Marko!!
